extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +76,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield macroTask();
```

Comment:
I agree we should yield before doing each check.

Here we try to test that loading ads count will decrease by 1 after 1 sec, and only after 1 sec. 
With yield promise, we can test that the ads count decrease by 1 after 1 sec. 
However with 
```
clock.tick(999);
expect(is3pThrottled(win)).to.be.true;
```
we can't tell if the promise is resolved only after 1sec. It is possible that the promise has already resolved after 999ms, but macroTasks not executed, loading ad count hasn't decresed by 1 when test code ran. 

This is probably right to do:
```
clock.tick(999);
yield macroTask();
expect(is3pThrottled(win)).to.be.true;
clock.tick(1);
yield promise;
expect(is3pThrottled(win)).to.be.false;
```